### PR TITLE
Restore CodeList

### PR DESCRIPTION
**Pull request by agoerler**

### CODE REVIEW

**Comment by beckermarc** [comment]
rebase on main?

File: db/books.cds [hunk 1]
```diff
@@ -47,10 +48,8 @@ annotate Authors with
 /**
  * Hierarchically organized Code List for Genres
  */
-entity Genres {
+entity Genres : sap.common.CodeList {
```
Comment (beckermarc) [inline db/books.cds:51]: @OlenaTi: What harm did the value help make, that's implicitly included in CodeList? If it did any harm I think the better approach would be to remove the annotation on `GenreHierarchy` by annotating it with `@cds.odata.valuelist: false`

**Review by beckermarc — APPROVED** [review]

**Comment by agoerler** [comment]
Before the merge I need to check if the value-help is still working ...

**Comment by agoerler** [comment]
> Before the merge I need to check if the value-help is still working ...

is working for me.

File: db/books.cds (hunk 1 above)
Comment (etimr) [reply db/books.cds:51]: you are right, it also works without `@cds.odata.valuelist: false` now: https://github.com/SAP-samples/cloud-cap-samples-java/pull/420

**Pull request merged**